### PR TITLE
Added predicted entities to bert_token_classifier_ner_btc

### PR DESCRIPTION
**Pull request by gadde5300**
## Description

Added "Predicted Entities" to bert_token_classifier_ner_btc.

## Motivation and Context

It fixes the issue [#763](https://github.com/JohnSnowLabs/streamlit-demo-apps/issues/763)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Code improvements with no or little impact
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [CONTRIBUTING](http://nlp.johnsnowlabs.com/contribute.html) page.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by murat-gunay — CHANGES_REQUESTED** [review]
the Predicted Entities section accepts only exact entity names separated by commas (,) without Person, Location or Organization or period (.) . Should be :

`PER`, `LOC`, `ORG`

**Review by murat-gunay — APPROVED** [review]

**Review by josejuanmartinez — APPROVED** [review]
It looks good to me. Sent it to Veysel for merging.

**Comment by vkocaman** [comment]
`uses the Deep Learning (torch) algorithm` >> do we really need to say this ?

**Comment by josejuanmartinez** [comment]
@vkocaman  thanks for spotting this out. I checked the changes only, not the original page. And there are several more things there:
1) `Description`: This model is trained on Broad Twitter Corpus (BTC) dataset, so that it can detect entities in Twitter-based texts successfully. It's based on bert_base_cased embeddings, which are included the model so, you don't need to use any embeddings components in the NLP pipeline.
2) Since we have this models in demo, can we use a better example with more entities? Dia didn't like the FUUULLHAM entity.
3) `data-set` : it's dataset.

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
@vkocaman thanks for spotting this out. I checked the changes only, not the original page. And there are several more things there:

Description: This model is trained on Broad Twitter Corpus (BTC) dataset, so that it can detect entities in Twitter-based texts successfully. It's based on bert_base_cased embeddings, which are included the model so, you don't need to use any embeddings components in the NLP pipeline.

Since we have this models in demo, can we use a better example with more entities? Sales people didn't like the FUUULLHAM entity.

data-set : it's dataset.

File: docs/_posts/murat-gunay/2021-09-09-bert_token_classifier_ner_btc_en.md [hunk 1]
```diff
@@ -17,9 +17,8 @@ use_language_switcher: "Python-Scala-Java"
 
 ## Description
 
-- This model is trained on Broad Twitter Corpus (BTC) data-set, so that it can detect entities in Twitter-based texts successfully.
-- `BertForTokenClassification()` module, which uses the Deep Learning (`torch`) algorithm, is used to train this model.
-- The embeddings `bert_base_cased` is embedded inside the model so, you don't need to use any embeddings in the NLP pipeline.
+- This model is trained on Broad Twitter Corpus (BTC) dataset, so that it can detect entities in Twitter-based texts successfully.
+- It's based on `bert_base_cased embeddings, which are included the model so, you don't need to use any embeddings components in the NLP pipeline.
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-09-09-bert_token_classifier_ner_btc_en.md:21]: Missing quote after `bert_bassed_cased`

File: docs/_posts/murat-gunay/2021-09-09-bert_token_classifier_ner_btc_en.md [hunk 2]
```diff
@@ -52,7 +51,7 @@ ner_converter = NerConverter()\
 pipeline =  Pipeline(stages=[documentAssembler, tokenizer, tokenClassifier, ner_converter])
 
 model = pipeline.fit(spark.createDataFrame(pd.DataFrame({'text': ['']})))
-test_sentences = ["""Wengers big mistakes is not being ruthless enough with bad players.""", """my dream FUUUUUULHAAAAAAM !!!.."""]
+test_sentences = ["""Wengers big mistakes is not being ruthless enough with bad players.""", """my dream Fulham F.C. !!!.."""]
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-09-09-bert_token_classifier_ner_btc_en.md:54]: Can you remove second example and use this one?

image

File: docs/_posts/murat-gunay/2021-09-09-bert_token_classifier_ner_btc_en.md [hunk 3]
```diff
@@ -69,7 +68,7 @@ val ner_converter = NerConverter()
 
 val pipeline =  new Pipeline().setStages(Array(documentAssembler, tokenizer, tokenClassifier, ner_converter))
 
-val data = Seq("Wengers big mistakes is not being ruthless enough with bad players.", "my dream FUUUUUULHAAAAAAM !!!..").toDF("text")
+val data = Seq("Wengers big mistakes is not being ruthless enough with bad players.", "my dream Fulham F.C. !!!..").toDF("text")
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-09-09-bert_token_classifier_ner_btc_en.md:71]: Can you remove second example and use this one?

image

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
Can you apply these changes? Thank youi!

**Review by josejuanmartinez — APPROVED** [review]
Looks good. Thank you.

File: docs/_posts/murat-gunay/2021-09-09-bert_token_classifier_ner_btc_en.md [hunk 4]
```diff
@@ -17,12 +17,12 @@ use_language_switcher: "Python-Scala-Java"
 
 ## Description
 
-- This model is trained on Broad Twitter Corpus (BTC) data-set, so that it can detect entities in Twitter-based texts successfully.
-- `BertForTokenClassification()` module, which uses the Deep Learning (`torch`) algorithm, is used to train this model.
-- The embeddings `bert_base_cased` is embedded inside the model so, you don't need to use any embeddings in the NLP pipeline.
+- This model is trained on Broad Twitter Corpus (BTC) dataset, so that it can detect entities in Twitter-based texts successfully.
+- It's based on `bert_base_cased` embeddings, which are included the model so, you don't need to use any embeddings components in the NLP pipeline.
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-09-09-bert_token_classifier_ner_btc_en.md:21]: which are included `in the model, so you don't need to use any embeddinsg component in the NLP pipeline`

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
This last thing and it's good to go.

**Review by josejuanmartinez — APPROVED** [review]
Looks good! Sending it to merge!

**Pull request merged**